src/Tools/dotnet-watch/test/AppWithDepsTests.cs

```diff
@@ -21,7 +21,7 @@ public AppWithDepsTests(ITestOutputHelper logger)
 
         [ConditionalFact]
         [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/23994")]
-        [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/23360", Queues = "Windows.10.Arm64;Windows.10.Arm64.Open;Debian.9.Arm64;Debian.9.Arm64.Open;(Debian.9.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-a12566d-20190807161036;(Debian.9.Arm64)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-a12566d-20190807161036")]
+        [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/23360", Queues = "Windows.10.Arm64;Windows.10.Arm64.Open;Debian.9.Arm64;Debian.9.Arm64.Open;(Debian.9.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-a12566d-20190807161036;(Debian.9.Arm64)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-a12566d-20190807161036;Windows.10.Arm64v8;Windows.10.Arm64v8.Open")]
```

Comment:
related to the trait idea, this would be a case where we could use SkipOnHelix(traits = "arm"), as opposed to listing every arm queue manually :/